fedora/client/bodhi.py

```diff
@@ -17,6 +17,10 @@
 # License along with python-fedora; if not, see <http://www.gnu.org/licenses/>
 #
 """
+DEPRECATED: This module is deprecated and has moved to bodhi.client.bindings.
+As the bodhi-0.9 client still uses it, it will remain here until Fedora 25 is
+EOL.
+
```

Comment:
I meant the python-fedora update, if we only push an update with the deprecation warning in rawhide, we should be fine :)